buildkite/bazelci.py

```diff
@@ -1670,8 +1677,13 @@ def main(argv=None):
                                    use_but=args.use_but,
                                    incompatible_flags=args.incompatible_flag)
         elif args.subparsers_name == "runner":
-            configs = fetch_configs(args.http_config, args.file_config)
-            execute_commands(config=configs.get("platforms", None)[args.platform],
+            tasks = get_tasks(args)
+            # there can be multiple tasks for a platform
+            for name in tasks:
+                task = tasks[name]
+                if hasattr(task, "platform") and task.platform != args.platform:
```

Comment:
What happens when a config has no explicit platform attribute and then this code runs on platform "ubuntu1404", but the platform (as specified in the name of the task, e.g. "macos") is incompatible?

Reply:
Fixed.